### ISSUE TITLE
inventory setup disappeared from sidebar after building runelite

### ISSUE THREAD

**Issue by Faustas156** [dillydill123/inventory-setups#139]
title, once i built runelite in intellij, the plugin disappeared and hasn't been able to come back, even after i removed it and put it back in again, if there is anyway to fix this, i would really like to know as this only affects inventory setups and nothing else

**Comment by dillydill123** [comment]
Are you cloning this repository and attempting to build?

**Comment by n0pe13** [comment]
Same issue as Faustas156. 

Logged in for the first time today around 1 hour ago. I turned on the Loot Tracker plugin. Noticed Loot Tracker "removed" Inventory Setup from the sidebar. Turned off Loot Tracker to see if Inventory Setup would reappear, nothing. Tried searching for Inventory Setup from the Configuration search menu, its not coming up. Inventory Setup is showing as installed from the Plugin Hub.
